tests/errors/e_cython_parallel.pyx

```diff
@@ -171,41 +175,54 @@ def bar():
         pass
 
 _ERRORS = u"""
-3:8: cython.parallel.parallel is not a module
-4:0: No such directive: cython.parallel.something
-6:7: cython.parallel.parallel is not a module
-7:0: No such directive: cython.parallel.something
-13:6: prange() can only be used as part of a for loop
-13:6: prange() can only be used without the GIL
-18:19: Invalid schedule argument to prange: invalid_schedule
-21:29: The parallel section may only be used without the GIL
-27:8: target may not be a Python object as we don't have the GIL
-30:9: Can only iterate over an iteration variable
-33:8: Must be of numeric type, not int *
-36:33: Nested parallel with blocks are disallowed
-39:12: The parallel directive must be called
-45:8: local variable 'y' referenced before assignment
-55:8: local variable 'y' referenced before assignment
-60:4: Reduction operator '*' is inconsistent with previous reduction operator '+'
-62:36: cython.parallel.parallel() does not take positional arguments
-65:36: Invalid keyword argument: invalid
-73:12: 'yield' not allowed in parallel sections
-77:16: 'yield' not allowed in parallel sections
-97:8: Cannot assign to private of outer parallel block
-98:8: Cannot assign to private of outer parallel block
-104:4: Reductions not allowed for parallel blocks
-110:6: local variable 'i' referenced before assignment
-119:14: Cannot read reduction variable in loop body
-121:19: prange() can only be used without the GIL
-121:20: stop argument must be numeric
-131:4: Memoryview slices can only be shared in parallel sections
-133:42: Must provide schedule with chunksize
-136:62: Chunksize must not be negative
-139:62: Chunksize not valid for the schedule runtime
-145:70: Calling gil-requiring function not allowed without gil
-149:33: Nested parallel with blocks are disallowed
-155:59: Calling gil-requiring function not allowed without gil
-158:57: Calling gil-requiring function not allowed without gil
-167:51: use_threads_if may not be a Python object as we don't have the GIL
-170:49: use_threads_if may not be a Python object as we don't have the GIL
+7:8: cython.parallel.parallel is not a module
+8:0: No such directive: cython.parallel.something
+10:7: cython.parallel.parallel is not a module
+11:0: No such directive: cython.parallel.something
+17:6: prange() can only be used as part of a for loop
+17:6: prange without releasing the GIL will only work well on freethreaded Python
+22:19: Invalid schedule argument to prange: invalid_schedule
+25:29: Parallel section without releasing the GIL will only work well on freethreaded Python
+31:8: target may not be a Python object as we don't have the GIL
```

Comment:
Arguable some of these restrictions could be relaxed (or the error message updated). But that's probably best done in a follow-up at some point